src/probability/kernel/basic.lean

```diff
@@ -400,6 +404,138 @@ end
+    have h_eq_sdiff : ∀ a, {b : β | (a, b) ∈ t'ᶜ} = set.univ \ {b : β | (a, b) ∈ t'},
+    { intro a,
+      ext1 b,
+      simp only [set.mem_compl_iff, set.mem_set_of_eq, set.mem_diff, set.mem_univ, true_and], },
+    simp_rw h_eq_sdiff,
+    have : (λ a, κ a (set.univ \ {b : β | (a, b) ∈ t'}))
+      = (λ a, (κ a set.univ - κ a {b : β | (a, b) ∈ t'})),
+    { ext1 a,
+      rw [← set.diff_inter_self_eq_diff, set.inter_univ, measure_diff],
+      { exact set.subset_univ _, },
+      { exact (@measurable_prod_mk_left α β _ _ a) t' ht', },
+      { exact measure_ne_top _ _, }, },
+    rw this,
+    exact measurable.sub (kernel.measurable_coe κ measurable_set.univ) h_meas, },
+  { -- we assume that the result is true for a family of disjoint sets and prove it for their union
+    intros f h_disj hf_meas hf,
+    have h_Union : (λ a, κ a {b : β | (a, b) ∈ ⋃ i, f i}) = λ a, κ a (⋃ i, {b : β | (a, b) ∈ f i}),
+    { ext1 a,
+      congr' with b,
+      simp only [set.mem_Union, set.supr_eq_Union, set.mem_set_of_eq],
+      refl, },
+    rw h_Union,
+    have h_tsum : (λ a, κ a (⋃ i, {b : β | (a, b) ∈ f i})) = λ a, ∑' i, κ a {b : β | (a, b) ∈ f i},
+    { ext1 a,
+      rw measure_Union,
+      { intros i j hij s hsi hsj b hbs,
+        have habi : {(a, b)} ⊆ f i, by { rw set.singleton_subset_iff, exact hsi hbs, },
+        have habj : {(a, b)} ⊆ f j, by { rw set.singleton_subset_iff, exact hsj hbs, },
+        simpa only [set.bot_eq_empty, set.le_eq_subset, set.singleton_subset_iff,
+          set.mem_empty_iff_false] using h_disj hij habi habj, },
+      { exact λ i, (@measurable_prod_mk_left α β _ _ a) _ (hf_meas i), }, },
+    rw h_tsum,
+    exact measurable.ennreal_tsum hf, },
+end
+
+lemma measurable_prod_mk_mem (κ : kernel α β) [is_s_finite_kernel κ]
+  {t : set (α × β)} (ht : measurable_set t) :
+  measurable (λ a, κ a {b | (a, b) ∈ t}) :=
+begin
+  rw ← kernel_sum_seq κ,
+  have : ∀ a, kernel.sum (seq κ) a {b : β | (a, b) ∈ t} = ∑' n, seq κ n a {b : β | (a, b) ∈ t},
+    from λ a, kernel.sum_apply' _ _ (measurable_prod_mk_left ht),
+  simp_rw this,
+  refine measurable.ennreal_tsum (λ n, _),
+  exact measurable_prod_mk_mem_of_finite (seq κ n) ht infer_instance,
+end
+
+lemma measurable_lintegral_indicator_const (κ : kernel α β) [is_s_finite_kernel κ]
+  {t : set (α × β)} (ht : measurable_set t) (c : ℝ≥0∞) :
+  measurable (λ a, ∫⁻ b, t.indicator (function.const (α × β) c) (a, b) ∂κ a) :=
+begin
+  simp_rw lintegral_indicator_const_comp measurable_prod_mk_left ht _,
+  exact measurable.const_mul (measurable_prod_mk_mem _ ht) c,
+end
+
+/-- For an s-finite kernel `κ` and a function `f : α → β → ℝ≥0∞` which is measurable when seen as a
+map from `α × β` (hypothesis `measurable (function.uncurry f)`), the integral
+`a ↦ ∫⁻ b, f a b ∂(κ a)` is measurable. -/
+theorem measurable_lintegral (κ : kernel α β) [is_s_finite_kernel κ]
+  {f : α → β → ℝ≥0∞} (hf : measurable (function.uncurry f)) :
+  measurable (λ a, ∫⁻ b, f a b ∂κ a) :=
```

Comment:
```suggestion
  measurable (λ a, ∫⁻ b, f a b ∂(κ a)) :=
```
I feel it makes things more readable. If you agree, can you change it at the various places you have `∂κ`? If you don't agree, you can ignore safely :-)

Reply:
I completely agree. I did it already in some places but I was not thorough about it.